.lintr

```diff
@@ -2,6 +2,8 @@ linters: with_defaults(
     object_length_linter = NULL,
     object_usage_linter = NULL,
     todo_comment_linter = NULL,
-    cyclocomp_linter = NULL
+    cyclocomp_linter = NULL,
+    trailing_whitespace_linter = NULL,
```

Comment:
You can make RStudio remove this for you automatically on save. In tools -> global options -> code -> saving -> strip trailing horizontal whitespace when saving